bin/resources/GameIndex.yaml

```diff
@@ -29301,6 +29306,15 @@ SLKA-25459:
   compat: 5
   gsHWFixes:
     cpuSpriteRenderBW: 2 # Fixes textures.
+    halfPixelOffset: 2 # Fixes misaligned bloom.
+    autoFlush: 2 # Fixes soft shadows.
+  patches:
```

Comment:
did you verify that this patch works this version?

Reply:
I think it does, I couldn't tell any difference between the NTSC-U and NTSC-K versions when playing them, memory addresses are also the same.

Reply:
i would prefer if that could get tested :)